packages/core/database/src/metadata/relations.ts

```diff
@@ -346,7 +384,9 @@ const createMorphOne = (
   attributeName: string,
   attribute: Relation.MorphOne,
   meta: Meta,
-  metadata: Metadata
+  metadata: Metadata,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  _options: MetadataOptions
```

Comment:
Why do we keep this here if we're not using it?

Reply:
It was left as a reminder that they're there and must be used for names, but I can remove them and maybe add a comment where it makes sense.

Reply:
Although it might be worth leaving for consistency, what do you think? There are only these two out of eight methods that aren't currently using them

Reply:
I'm a bit annoyed to see arguments we don't need being declared, but it's not the biggest deal